src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/ActionRuleEndpoint.java

```diff
@@ -302,4 +306,38 @@ private CollectionExercise getCollectionExercise(UUID uuid, String createdBy)
                   HttpStatus.NOT_FOUND, "Collection exercise not found");
             });
   }
+
+  @GetMapping(value = "/caseCount")
+  public Integer getActionRuleCaseCount(
+      @RequestParam(value = "actionRuleId") UUID actionRuleId,
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    ActionRule actionRule =
+        actionRuleRepository
+            .findById(actionRuleId)
+            .orElseThrow(
+                () -> {
+                  log.with("actionRuleId", actionRuleId)
+                      .with("httpStatus", HttpStatus.NOT_FOUND)
+                      .with("userEmail", userEmail)
+                      .warn("Failed to dry run action rule, Action Rule not found");
+                  return new ResponseStatusException(HttpStatus.NOT_FOUND, "Action Rule not found");
+                });
+
+    userIdentity.checkUserPermission(
+        userEmail,
+        actionRule.getCollectionExercise().getSurvey(),
+        UserGroupAuthorisedActivityType.LIST_ACTION_RULES);
+
+    StringBuilder query = new StringBuilder();
+    query
+        .append(
+            String.format(
+                "SELECT COUNT(*) FROM casev3.cases WHERE collection_exercise_id='%s'",
+                actionRule.getCollectionExercise().getId().toString()))
+        .append(" AND ")
+        .append(actionRule.getClassifiers());
```

Comment:
The query is always expecting a classifier which isn't always the case (I know in production we'd have classifiers but for dev work it might not be the case)